ch01-getting-started.asc

```diff
@@ -21,6 +21,6 @@ include::book/01-introduction/sections/help.asc[]
 
 === Zusammenfassung
 
-Sie sollten jetzt ein grundlegendes Verständnis dafür haben, was Git ist und wie es sich von anderen zentralisierten Versionsverwaltungs-Systemen unterscheidet, die Sie ev. bisher eingesetzt haben.
+Sie sollten jetzt ein grundlegendes Verständnis dafür haben, was Git ist und wie es sich von anderen zentralisierten Versionsverwaltungssystemen unterscheidet, die Sie ev. bisher eingesetzt haben.
```

Comment:
Im Deutschen neigen wir leider zu langen, zusammengesetzten Wörtern.
Um das Verständnis beim Lesen zu verbessern, bevorzuge ich eigentlich die Variante mit dem Bindestrich, auch wenn sie, streng orthografisch, vielleicht nicht ganz korrekt ist.

Reply:
Hier bin ich mit beidem einverstanden. Wichtig ist m.E. nur, dass es *einheitlich* ist. Es gab 1x "Versionsverwaltungs-System" und 13x "Systemverwaltungssystem". Deshalb habe ich nur die eine Instanz geändert. Ich schlage vor, dass wir es erst einmal so lassen. Mal sehen, was ich noch für zusammengesetzte lange Worte finde und wie diese behandelt werden. Einverstanden?

Reply:
Ich denke nicht, dass man hier vereinheitlichen _muss_. Wir sollten, von Fall zu Fall, die Variante wählen, die es ermöglicht den Text beim schnellen Lesen besser zu erfassen.
Wie gesagt, ist mir persönlich die Variante hier **mit** dem Bindestrich lieber. Da aber beide Formen nebeneinander existieren, kann ich mit beiden leben.

Reply:
An dieser einen Stelle kannst du also "Versionsverwaltungs-System" schneller lesen als "Systemverwaltungssystem"?

Argumente, die ich hätte gelten lassen wären
- eines der beiden Worte sollte betont werden (hätte man aber auch mit Kursivauszeichnung erreichen können) oder
- die Variante mit Bindestrich sorgt für einen besseren Umbruch in der PDF-Ausgabe (wenn dem aber so wäre, dann würde ich noch einen Kommentar in der Quelle hinterlassen).

Wie ich ebenfalls schon geschrieben habe, wäre ich auch damit einverstanden *alle* 14 Vorkommen durch die Variante mit Bindestrich zu ersetzen.

Haben wir eine Schiedsrichter-/Entscheidungsmeinung?

Reply:
> An dieser einen Stelle kannst du also "Versionsverwaltungs-System" schneller lesen als "Systemverwaltungssystem"?

Nein, ich meinte, beim schnellen Lesen (überfliegen) des Textes ist es besser zu erfassen.
Aber, wie schon gesagt, das ist wirklich nur meine persönliche Meinung.

Reply:
Da wir hier wohl einen Mehrheitsentscheid brauchen: mir gefällt `Versionsverwaltungssystem` besser.

Reply:
OK